scripts/flash/finalize-market.js

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var chalk = require("chalk");
+var getTime = require("./get-timestamp");
+
+function finalizeMarketInternal(augur, marketID, auth, callback) {
+  var marketPayload = { tx: { to: marketID } };
+  augur.api.Market.getFeeWindow(marketPayload, function (err, feeWindowAddress) {
+    if (err) { console.error(err); callback(err); }
+    if (feeWindowAddress === "0x0000000000000000000000000000000000000000") {
+      console.log(chalk.red("Market has not be reported on, need initial-report before finalization"));
+      callback(null);
```

Comment:
Do you intend for the function to continue executing after `callback` is called?  If not, should use `return callback(null);`.